pkg/utils/k8sutil.go

```diff
@@ -56,8 +56,6 @@ import (
 
 const (
 	pubsubFunc     = "PubSub"
-	busybox        = "busybox@sha256:be3c11fdba7cfe299214e46edc642e09514dbb9bbefcd0d3836c05a1e0cd0642"
```

Comment:
What happened to this image ? honestly I don't remember what it is/was used for

Reply:
As per feedback from @andresmgot  this is not used anymore so I removed it. Also `golint` does not warn about unused `const`